packages/gatsby/src/utils/babel-parse-to-ast.ts

```diff
@@ -43,7 +43,7 @@ const PARSER_OPTIONS = {
   ],
 }
 
-export function getBabelParserOptions(filePath: string) {
+export function getBabelParserOptions(filePath: string): object {
```

Comment:
instead of object can we do `typeof PARSER_OPTIONS`?

Reply:
Unfortunately, It is impossible because of the second argument type of `parser.parse` method is already declared `parser.ParserOptions` | `undefined` by `@babel/parser`